activities/communication/index.md

```diff
@@ -7,39 +7,23 @@ type: Index
 These are all the activities, documents and templates for Communication.
 
 * [Communication principles](communication-principles.md)
+* [Communications plan](communications-plan.md)
 * [Using chat.publiccode.net](using-chat.md)
-* [Community discussions](community-discussions.md)
-* [Instructions on how to run a community call](running-community-call.md)
-* [Phone numbers](phone-numbers.md)
-
-Community calls are a way to get direct input from the community on topics that in any way are of concern to them. We do them on a bi-weekly schedule for regularity and they are open for anyone to join.
-
-## Using and managing phone numbers
+* [How and why to write a blogpost](blogging.md)
+* [Email signature template](email-signature-template.md)
 
```

Comment:
I'd also add other activities related to Communications such as attending events, wether as an organizer, speaker or as an attendee, Social Media, webinars or workshops we might offer for free

Reply:
They could also be part of the communications plan (I have left a comment there), what do you guys think?

Reply:
I've added a link to events to the index page.

Reply:
We currently don't have separate guidance for social media or webinars, so I can't add this now - but if we have such things in the future, then sure. I've also added a link to workshops.